src/test/java/org/jenkinsci/plugins/workflow/ArtifactManagerTest.java

```diff
@@ -401,7 +402,7 @@ private void assertFile(VirtualFile f, String contents) throws Exception {
             assertEquals(contents.length(), f.length());
             assertThat(f.lastModified(), not(is(0)));
             try (InputStream is = f.open()) {
-                assertEquals(contents, IOUtils.toString(is));
+                assertEquals(contents, IOUtils.toString(is, Charset.defaultCharset()));
```

Comment:
Use UTF-8 here. (Should be irrelevant, since the Unicode tests we are doing involve file _names_ not contents, but as a matter of style we try to use UTF-8 unless there is a specific reason like external process output why we would expect something to be in another encoding.)